lib/helpers.js

```diff
@@ -40,14 +43,39 @@ function randomTweets(tweets, limit) {
     return retTweets;
 }
 
+function cssWithBuster(uri, sri) {
+    return '<link rel="stylesheet" href="' + uri + '?' + cacheBuster +
+        '" integrity="' + sri + '" crossorigin="anonymous" />';
+};
+
+function jsWithBuster(uri, sri) {
+    return '<script src="' + uri + '?' + cachBuster +
+        '" integrity="' + sri + '" crissorigin="anonymous"';
+};
+
+function imgWithBuster(uri, attributes) {
+    var attrs = '';
+
+    for(var key in attributes) {
+        attrs += ' ' + key + '="' + attributes[key] + '"';
+    }
+
+    return '<img src="' + uri + '?' + cacheBuster + '"' + attrs + ' />';
```

Comment:
No need to use `/>`. It's already self closing in HTML5, The same applies for `link`.